doc/src/build/zk_login.md

```diff
@@ -2,23 +2,24 @@
 title: About zkLogin
 ---
 
-zkLogin is a Sui primitive that lets wallets and apps link Sui objects with an OAuth credential—enabling users to perform transactions using both the credential and attached objects.
+zkLogin is a Sui primitive that let users send transactions from a Sui address using an OAuth credential, without publicly linking the two.
 
-zkLogin eliminates the need for users to handle private keys or recall mnemonics/passwords in wallets. By guiding users through the OAuth flow, an zkLogin address is generated, enabling any associated objects to execute transactions on-chain.
+zkLogin is designed with the following goals in mind:
 
-zkLogin is designed with the following principles in mind:
+1. **Simple Onboarding**: zkLogin allows users transact on Sui using the familiar OAuth login flow, eliminating the friction of handling cryptographic keys or remembering mnemonics.
 
-1. **Simple Onboarding**: With the familiar OAuth login flow, zkLogin enables users to easily create a Sui wallet and engage with Sui—eliminating the need for cryptographic key management.
+1. **Fully Self-Custodial**: A zkLogin transaction requires user approval via the standard OAuth login process--the OAuth provider cannot transact on the user's behalf.
 
-1. **Fully Self-Custodial**: A zkLogin transaction requires user approval within the OAuth login process, preventing unilateral fund movement from a zkLogin wallet—even by the OAuth provider.
+1. **Secure**: zkLogin is a two-factor authentication scheme: sending a transaction requires both a credential from a recent OAuth login and a password not managed by the OAuth provider. An attacker who compromises an OAuth account cannot transact from the user's corresponding Sui address unless they separately compromise the password.
```

Comment:
we generally don't use the word password (which is typically something user generated and with high probability lower entropy than 128 bits), instead we use the wording: high entropy salt per account.

Reply:
This is probably a broader conversation, but I I think it's really important to use a familiar term like "password" even if from a cryptography perspective "high entropy salt" is the precise description. IMO we need a commonsense term that captures how devs/users should think about and store these.

Reply:
This is probably a broader conversation, but I I think it's really important to use a familiar term like "password" even if from a cryptography perspective "high entropy salt" is the precise description. IMO we need a commonsense term that captures how devs/users should think about, store, and describe these.

Reply:
I'll use "salt" for now pending a broader discussion